ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -191,6 +192,7 @@ Result conv2d(
 
         if (bypass_halo) {
             if (input_tensor_post_tm.layout() == Layout::TILE) {
+                input_tensor_post_tm = ttnn::reshape(input_tensor_post_tm, input_tensor_post_tm.get_padded_shape());
```

Comment:
why do we need this reshape here?

Reply:
to_layout was not using the padded shape, and was instead using the logical shape.  This was causing an error.

Reply:
at least we should log an issue with unit tests and reference it here.
Would be better if we can just fix to layout.

Halo with untialize is doing this properly?

Reply:
Created an issue for this : https://github.com/tenstorrent/tt-metal/issues/16330